go/vt/tabletserver/tabletenv/tabletenv.go

```diff
@@ -29,10 +29,27 @@ var (
 	WaitStats = stats.NewTimings("Waits")
 	// KillStats shows number of connections being killed.
 	KillStats = stats.NewCounters("Kills", "Transactions", "Queries")
-	// InfoErrors shows number of various non critical errors happened.
-	InfoErrors = stats.NewCounters("InfoErrors", "Retry", "DupKey")
 	// ErrorStats shows number of critial erros happened.
-	ErrorStats = stats.NewCounters("Errors", "Fail", "TxPoolFull", "NotInTx", "Deadlock", "Fatal")
+	ErrorStats = stats.NewCounters(
+		"Errors",
```

Comment:
What is "Errors"? A total of all values?

In general, it's not advised to have a total in your map as well. Instead, the monitoring system should figure this out for you. Maybe ask Javeen on his opinion on this?